interviews/frontend-takehome.md

```diff
@@ -39,19 +39,17 @@ Non-core features encouraged for completion after the core features.
 - 👍 3: Requested delighter mostly completed or completed with a few minor bugs/flaws
 - 👎 2: Little to no delighter content was completed
 
-## Frontend Specifics
```

Comment:
Line 17 has a link to this line. I think we may want to keep the heading?

Reply:
oh interesting, but frontend specifics doesn't make sense to me... does it make sense to you?

Reply:
I think the intent is to give further explanation around what we look for or don't look for for frontend knowledge.

Maybe we could rename it to `Frontend Knowledge Specifics`

Reply:
hmm ok i guess it's fine as is